spec/requests/todo_lists_spec.rb

```diff
@@ -1,7 +1,129 @@
 require 'rails_helper'
 
 RSpec.describe "TodoLists", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+  let!(:user) {create(:user)} #Create users 
+  let!(:todo_lists) { create_list(:todo_list, 5, user: user) } # Create 5 todo lists for the user
+  let(:auth_headers) { user.create_new_auth_token } # Authentication headers
+
+
+  def json
+    JSON.parse(response.body)
+  end
```

Comment:
is this method really required

Reply:
its not required as i have updated in my code already but its not getting shown . so without this piece of code is working.